tests/main/lxd/task.yaml

```diff
@@ -15,13 +14,26 @@ kill-timeout: 25m
 # Start before anything else as it can take a really long time.
 priority: 1000
 
+environment:
+    REFRESH_APP_AWARENESS_OUTER/snapd_cgroup_just_inside: false
+    REFRESH_APP_AWARENESS_INNER/snapd_cgroup_just_inside: true
+    REFRESH_APP_AWARENESS_OUTER/snapd_cgroup_just_outside: true
+    REFRESH_APP_AWARENESS_INNER/snapd_cgroup_just_outside: false
+    REFRESH_APP_AWARENESS_OUTER/snapd_cgroup_both: true
+    REFRESH_APP_AWARENESS_INNER/snapd_cgroup_both: true
+    REFRESH_APP_AWARENESS_OUTER/snapd_cgroup_neither: false
+    REFRESH_APP_AWARENESS_INNER/snapd_cgroup_neither: false
+
 prepare: |
     # using apt here is ok because this test only runs on ubuntu
     echo "Remove any installed debs (some images carry them) to ensure we test the snap"
     # apt -v to test if apt is usable (its not on ubuntu-core)
     if command -v apt && apt -v; then
         apt autoremove -y lxd
     fi
+    # Depending on test variant, enable refresh-app-awareness outside to test
+    # the interaction between lxd and cgroup at /run/snapd/cgroup
```

Comment:
aren't we not using `/run/snapd/cgroup` anymore? What's the utility of this test if we aren't using /run/snapd/cgroup anymore?

Reply:
This is a stale comment. I'll fix it.